tests/test_utils.py

```diff
@@ -125,13 +125,13 @@ def autonomy_fs(test_packages_filesystem):
 
 def test_get_paths_changed_only(test_packages_filesystem):
     """Test get_paths."""
-    assert test_packages_filesystem == str(Path.cwd())
+    assert Path.cwd() == test_packages_filesystem
```

Comment:
but why lol

Reply:
the old `isolated_filesystem` contextmanager:
1. `temp_dir_path = Path(temp_dir).resolve()`
2. `yield str(Path(temp_dir_path))`
and then in all tests, compared equal to:
3. `str(Path.cwd())`

pretty horrendous no? Just keep `Path` like a sane person

Reply:
ah i see you are saying i was calling string on the type aha